main.go

```diff
@@ -895,11 +896,49 @@ func calculateLinesTaken(u *User, s string, width int) {
 }
 
 // bansContains reports if the addr or id is found in the bans list
-func bansContains(b []Ban, addr string, id string) bool {
+func bansContains(b []Ban, addr string, id string) bool { 
 	for i := 0; i < len(b); i++ {
 		if b[i].Addr == addr || b[i].ID == id {
 			return true
 		}
 	}
 	return false
 }
+
+func logUserActivity(user *User, action string, loggingEnabled bool) {	
+    if !loggingEnabled { false //logging on Devzat is disabled by default, to enable it please switch the statement from false to true 
```

Comment:
for the error on line 909, because logging is disabled by default, (untyped bool constant) is not used is not an issue